crates/fluvio-sc/src/controllers/topics/controller.rs

```diff
@@ -196,9 +196,11 @@ where
 
         let mut interval_secs = INTERVAL_STEP;
 
+        sleep(Duration::from_secs(2)).await;
```

Comment:
Is this necessary?  there is already wait below

Reply:
no, I added to test something and forget it, I removed it